calibre-web.json

```
@@ -0,0 +1,59 @@
+{
+  "Calibre Web": {
+    "description": "Calibre-Web is a web app providing a clean interface for browsing, reading and downloading eBooks using an existing Calibre database.<p>The Web interface allows user logins and the management and download of e-Books through a clean web interface.</p><p>For more information please see <a href='https://github.com/janeczku/calibre-web' target='_blank' rel='noopener noreferrer'>the oficial Github page</a> and <a href='https://hub.docker.com/r/linuxserver/calibre-web' target='_blank' rel='noopener noreferrer'>the Docker container page.</a></p></br>If linking to the Calibre Rockon use the directory '/config/Calibre Library' as your database location during configuration in the WebUI.",
+    "more_info": "",
```

I would add more information on shares selection, for instance, as well as first login information:
```suggestion
    "more_info": "<h4>Using Calibre-web in combination with the Calibre Rock-On</h4><p>Calibre-web needs access to your E-books library served by the Calibre Rock-On. To do so, during the Calibre-web Rock-On installation wizard, make sure to choose for <em>Media Storage</em> the same share as the one chosen for the Calibre Rock-on.</p><h4>First login configuration</h4><p>On the initial Calibre-web setup screen, enter <code>/books</code> as your calibre library location.</p><p>The default admin login username is <code>admin</code>, and its password is <code>admin123</code>.</p>",
```

---

I have modified this suggestion to better explain the process and have incorporated it.